bitcoin/src/blockdata/transaction.rs

```diff
@@ -1256,8 +1256,8 @@ const fn predict_weight_internal(
     // version:
         4 +
     // count varints:
-        VarInt(input_count as u64).size() +
-        VarInt(output_count as u64).size() +
+        encode::varint_size_const(input_count as u64) +
+        encode::varint_size_const(output_count as u64) +
```

Comment:
In e9123d1d25bce9be4d1702ae9da6b99bd2e9674e:

These can use `.into_u64`. (Similar below and in several other places.)